lib/services/rally/rally_resource.rb

```diff
@@ -80,6 +80,10 @@ def maybe_workspace
     end
   end
 
+  def rally_workspace_url
+    "#{API_URL}/workspace/#{@service.data.workspace}"
```

Comment:
Is there anywhere in the Rally docs that says that you need to have the full url when specifying the workspace?